### ISSUE TITLE
Will this work with RMD-L-12025

### ISSUE THREAD

**Issue by Matt-Nor** [ThuanLeUte/RMD_X8_PRO_CAN_BUS_CONTROLER#3]
Hello,

I found your project through the help form on the MyActuator form.

I'm trying to control a RMD-L-12025 using this code, and Arduino & Seeed's Can Bus shield. When I open the serial monitor to send commands it seems to be be communicating properly with the either the motor or the Arduino, can you advise?

Best,
Matt

**Comment by ThuanLeUte** [comment]
Thank you for your interested. This code can support the motor that you mention. Can you explain what issue that you are facing now ?

**Comment by Matt-Nor** [comment]
I don't believe it's writing commands to the motor properly. I am using a different CAN chip hat this is build for as the shield is out of stock. As a result I'm having to use a slightly different library and haven't been able to make things work as of yet. 

I've attached my code along the library I'm using if you're able to provide any insight. 
[nanoMotorControl.zip](https://github.com/ThuanLeUte/RMD_X8_PRO_CAN_BUS_CONTROLER/files/8319874/nanoMotorControl.zip)

**Comment by ThuanLeUte** [comment]
I think if you want to make the thing work. Try to use my code and the right board. For the code, I have tested on the real board.
If you want to have a quick support of want me to do the project, you can directly contact me via WhatApp: +84379523140
